### PR TITLE
Admin Cookies are now entropy immune

### PR DESCRIPTION
**Pull request by DeityLink**
:cl:
* tweak: Cookies spawned by admin in return for prayers are now blessed with a protection from thermal entropy, and will no longer freeze in space or burn in an inferno.

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -2833,7 +2833,12 @@
 			spawntype = /obj/item/weapon/coin/pumf
 			feedback = "You have greatly angered the gods, and their grudge toward you has been crystalized into a damned pumf coin."
 
-		if(!H.put_in_hands( new spawntype(H)))
+		var/obj/item/reward = new spawntype(H)
+		if (answer == "Cookie")
+			var/obj/item/weapon/reagent_containers/food/snacks/cookie/C = reward
+			C.thermal_variation_modifier = 0
+
+		if(!H.put_in_hands(reward)
```
Comment (west3436) [inline code/modules/admin/topic.dm]:
```suggestion
		if(!H.put_in_hands(reward))
```

**Review by west3436 — CHANGES_REQUESTED** [review]
Linters

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/modules/admin/topic.dm]: 0% tested

**Review by west3436 — APPROVED** [review]

**Comment by SonixApache** [comment]
as someone who received a frozen cookie in response to a cry for help in the void once, i thank you

**Comment by jwhitak** [comment]
I personally find it hilarious to give a cookie to someone dying in a fire or freezing in space only for the cookie to bust into flames/ash or freeze solid. Perfect for the slapstick humor of the station.

Ah well, rip

**Pull request merged**